streams/src/main/java/org/apache/kafka/streams/state/Stores.java

```diff
@@ -230,10 +230,9 @@ private static WindowBytesStoreSupplier persistentWindowStore(final String name,
     /**
      * Create a persistent {@link SessionBytesStoreSupplier}.
      * @param name              name of the store (cannot be {@code null})
-     * @param retentionPeriod   length ot time to retain data in the store (cannot be negative)
-     *                          Note that the retention period must be at least long enough to contain the
-     *                          windowed data's entire life cycle, from window-start through window-end,
-     *                          and for the entire grace period.
+     * @param retentionPeriod   length to time to retain data in the store (cannot be negative)
```

Comment:
This doesn't read well.  Assuming it needs to be as long as `inactivity-gap` plus `grace-period` then the following reads better to me:

```suggestion
     * @param retentionPeriod   length of time to retain data in the store (cannot be negative)
     *                          Note that the retention period must be at least as long as
     *                          inactivity-gap plus grace-period.
```